style.css

```diff
@@ -3,37 +3,59 @@
   font-family: sans-serif;
   letter-spacing: 2px;
 }
+html {
+  font-size: 15px;
+}
 body {
   background-color: #e0e0e0;
 }
 header {
-  position: fixed;
+  position: initial;
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: center;
   width: 100%;
   margin-left: 0;
   top: 0;
   left: 0;
   z-index: 10;
-  padding: 20px;
+  padding: 10px;
+}
+h1 {
+  justify-content: center;
+  display: flex;
+  font-size: 2rem;
+}
+body {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
 }
 .card-section {
   display: flex;
   flex-wrap: wrap;
   width: 100%;
-  flex-direction: column;
   max-width: fit-content;
-  margin-left: auto;
-  margin-right: auto;
-  padding: 0 20px;
+  overflow: anywhere;
```

Comment:
firefox (which i'm using) does not like this.

page seems to display fine so it is not needed